src/code.c

```diff
@@ -1806,6 +1806,17 @@ void CodeStringExpr (
     PushExpr( string );
 }
 
+void CodePragma (
+    Obj              pragma )
+{
+    GAP_ASSERT(IS_STRING_REP(pragma));
+
+    Expr pragmaexpr = NewStat( T_PRAGMA, sizeof(UInt) );
+    Int ix = PushValue(pragma);
+    WRITE_EXPR(pragmaexpr, 0, ix);
+    PushStat( pragmaexpr );
+}
```

Comment:
This is completely new code, so you are free to clang-format it :-)

Reply:
Done.